File: codes/fibo/fibo.rb
#!/usr/bin/ruby

def fibo(n)
	if (n <= 1)
		return n
	else 
		return fibo(n-1) + fibo(n-2)
	end
end

print "fibo ",ARGV[0],":", fibo(Integer( ARGV[0]))
print "\n"

File: codes/fibo/fibo.pl
#!/usr/bin/perl



sub fibo {
	my($n) = $_[0];
	if( $n <2){
		return($n);}
	else{
		return fibo($n-1)+fibo($n-2);
	}

}
$fibon =  fibo(int($ARGV[0]));
print "fibo $ARGV[0]: $fibon\n" ;



